### ISSUE TITLE
Conflict with Metadata Browse module

### ISSUE THREAD

**Issue by HHeimlich** [hafizchin/DomainManager#9]
Hello everybody and thanks to hafizchin.

I have been working with DomainManager since last week and I recently discovered a conflict between DomainManager and Metadata Browse modules.

The latest module adds a link to same results for metadata values, in my case this is the configuration:

![metadatabrowseConfig](https://user-images.githubusercontent.com/4049585/101582601-eacab280-39d2-11eb-9d25-5f9fc0208a98.png)

My ambience: 

Debian 9
Omeka-S 2.1.2
Domain Manager 1.2 activated
Apache 2.4.25
Metadata Browse 1.3.0
(https://omeka.org/s/modules/MetadataBrowse/)

Before DomainManager or if it is switched off, my item page looks like this:

![correctLink](https://user-images.githubusercontent.com/4049585/101583368-44cb7800-39d3-11eb-9ca4-6560c0e271fc.png)

Please note the URL bar and, in the bottom left corner, the link formed with my URL slug and, in the latest, the _item_ page with the query string. The result page is what I expect, a list of values matching the criteria.

If I turn DomainManager on the result page is my Home page.

![faiLink](https://user-images.githubusercontent.com/4049585/101583766-1306e100-39d4-11eb-91a5-2f7b005d44a2.png)

Please note the lack of my URL slug both in URL bar and in the link formed in bottom left corner. In the second case the _item_ page disappear with the URL slug and then, when I click the link, the result page is my Home receiving the search's query string.

What can I do?

Thanks in advance.

**Comment by HHeimlich** [comment]
I suppose it isn't the best way to do it, but I have corrected the issue adding to .htaccess the net lines:

RewriteCond %{THE_REQUEST} \?Search [NC]
RewriteCond %{THE_REQUEST} !item
RewriteRule ^ /item [QSA,R,L]

I hope it'll help somebody.

**Comment by hafizchin** [comment]
Hi,

I have deployed some changes to the code. Let me know if this fixes the issue without modifying .htaccess.

**Comment by HHeimlich** [comment]
Hi,

Sorry for the late response. I have tried and I got the message in the attached image. What I do first was to deactivate Domain Manager and then update it.

I also tried to deactivate Archive Repertory 3.15.9 and then update Domain Manager but it's the same.

Another option was to update Archive Repertory to 3.15.11.3 but it's not compatible with Omeka-S 2.1.2

![Archive Repertory 3 15 9](https://user-images.githubusercontent.com/4049585/108101051-a1f04380-707e-11eb-8b05-925dc406bba9.png)

Thanks a lot, I'll be watchful.
